backend/src/main/kotlin/app/ehrenamtskarte/backend/userdata/webservice/UserImportHandler.kt

```diff
@@ -0,0 +1,113 @@
+package app.ehrenamtskarte.backend.userdata.webservice
+
+import app.ehrenamtskarte.backend.common.utils.dateStringToStartOfDayInstant
+import app.ehrenamtskarte.backend.projects.database.ProjectEntity
+import app.ehrenamtskarte.backend.projects.database.Projects
+import app.ehrenamtskarte.backend.userdata.database.UserEntitlementsRepository
+import app.ehrenamtskarte.backend.userdata.exception.UserImportException
+import io.javalin.http.Context
+import org.apache.commons.csv.CSVFormat
+import org.apache.commons.csv.CSVParser
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.time.Instant
+import java.time.ZoneId
+import java.time.format.DateTimeParseException
+
+class UserImportHandler {
+
+    private val logger: Logger = LoggerFactory.getLogger(UserImportHandler::class.java)
+
+    fun handle(ctx: Context) {
```

Comment:
Probably we should throw an error if we're on production for now.

Reply:
Done